eclipse-theia/theia
# Make behaviour of "Open Workspace/Folder" consistent with VS Code

#### What it does
The PR renames the "Open Workspace" file menu to "Open Workspace from File" and only allows selection of files in the resulting dialog on all platformas and browser/electron.

Fixes #12534

Contributed on behalf of STMicroelectronics

#### How to test
Use the "Open Workspace from File..." menu and make sure the behavior is as described above acrosss all platforms and electron/browser.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
